### DISCUSSION TITLE
Additional settings for the "Update Branch" option in PRs

### DISCUSSION THREAD

**Discussion by edkimmel** [community/community#36804] [Pull Requests]
<img width="810" alt="Screen Shot 2022-10-20 at 11 41 32 AM" src="https://user-images.githubusercontent.com/3375008/196995806-18f7e387-258d-422a-bdb3-9322159b419e.png">
In addition to allowing this feature to be turned on and off per repository:

- Allow us to designate merge or rebase to be displayed as the default option
- Allow us to disallow merge or rebase as an option

**Comment by ossanna16** [comment]
Thank you for asking a good question @edkimmel, I'm glad you found our community 🙂

**Comment by jdeamicis** [comment]
Duplicate of https://github.com/orgs/community/discussions/12032 ?
